test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -320,6 +320,73 @@ var _ = Describe("Mixedcpus", Ordered, func() {
 				// in the child processes
 			})
 		})
+		When("trying to deploy a burstable pod", func() {
+			It("should cause the burstable pod to fail", func() {
+				By("creating a burstable pod")
+				rl := &corev1.ResourceList{
+					corev1.ResourceCPU: resource.MustParse("1"),
+					sharedCpusResource: resource.MustParse("1"),
+				}
+				_, err := createPod(ctx, testclient.Client, testutils.NamespaceTesting,
+					withRequests(rl),
+					withLimits(rl),
+					withRuntime(components.GetComponentName(profile.Name, components.ComponentNamePrefix)))
+				Expect(err.Error()).To(ContainSubstring("resource but pod is not Guaranteed QoS class"))
```

Comment:
this (substring in error message) is an antipattern but I don't have a quick fix handy so let's treat like technical debt and fix "later" ( :\ ).